apps/portal/app/routes/readonly+/identity+/$id.tsx

```diff
@@ -66,28 +71,60 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
     logger('Failed to fetch list:', error)
   }
 
-  let vaultDetails: VaultDetailsType | null = null
+  let atomResult: GetAtomQuery | null = null
```

Comment:
I came up with a more optimal pattern for this when working on Data Created. Let's leave this as it is since it works and then we can see if it's worth doing a pass to update these all at once!

Basically -- since this is using dehydrated state we can actually grab it from the parent loader. I want to test a bit more and then see what we think